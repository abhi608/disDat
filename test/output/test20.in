Site: 1 failed
Site: 2 failed
Site: 4 failed
Site: 6 failed
Site: 8 failed
Site: 10 failed
Starting T1
T1 is waiting on x3
T1 got read lock on x4 having value 40
T1 is waiting on x5
Site: 4 recovered
Site: 6 recovered
Though the site holding x3 is recovering, T1 got read lock on x3 having value 30 since its the only copy
Though the site holding x5 is recovering, T1 got read lock on x5 having value 50 since its the only copy
T1 read the value 50 of variable x5
T1 read the value 30 of variable x3
T1 read the value 40 of variable x4
T1 committed
DEBUG| Clearing site locks for T1 variable: x3
Clearing locks for T1 variable: x3
DEBUG| Clearing site locks for T1 variable: x4
Clearing locks for T1 variable: x4
DEBUG| Clearing site locks for T1 variable: x5
Clearing locks for T1 variable: x5
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
All other variables have same initial value
=== Site 4 ===
x20 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
x20 is not available for reading
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x10 is not available for reading
x15: 150 (available at site 6 for reading as it is the only copy or has been written after recovery)
x5: 50 (available at site 6 for reading as it is the only copy or has been written after recovery)
x4 is not available for reading
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
This site is down
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
This site is down
