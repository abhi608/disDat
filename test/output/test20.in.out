Site: 1 failed
Site: 2 failed
Site: 4 failed
Site: 6 failed
Site: 8 failed
Site: 10 failed
Starting T1
T1 is waiting on x3
T1 got read lock on x4 having value 40
T1 is waiting on x5
Site: 4 recovered
Site: 6 recovered
Though the site holding x3 is recovering, T1 got read lock on x3 having value 30 since its the only copy
Though the site holding x5 is recovering, T1 got read lock on x5 having value 50 since its the only copy
T1 read the value 50 of variable x5
T1 read the value 30 of variable x3
T1 read the value 40 of variable x4
T1 committed
DEBUG| Clearing site locks for T1 variable: x3
Clearing locks for T1 variable: x3
DEBUG| Clearing site locks for T1 variable: x4
Clearing locks for T1 variable: x4
DEBUG| Clearing site locks for T1 variable: x5
Clearing locks for T1 variable: x5
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 20 at site 3
=== Site 4 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x10 is not available for reading
x6 is not available for reading
x4 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 20 at site 5
=== Site 6 ===
x20 is not available for reading
x14 is not available for reading
x15: 150 (available at site 6 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x8 is not available for reading
x6 is not available for reading
x16 is not available for reading
x5: 50 (available at site 6 for reading as it is the only copy or has been written after recovery)
x4 is not available for reading
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 20 at site 7
=== Site 8 ===
This site is down
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 20 at site 9
=== Site 10 ===
This site is down
